test/controllers/export_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+# Controller tests for info pages
+class ExportControllerTest < FunctionalTestCase
+  def setup
+    @controller = ::ExportController.new
```

Comment:
Weird!  So this is what you need to add to this test for it to get the right controller??

Reply:
That was a suggestion on SO. But the constant `ExportController` comes up uninitialized... and anyway the setup block shouldn't be necessary. 

Car was repaired, i'm headed back out. If you figure out what is going on here, please commit it and merge to the observer controller refactor!